src/ansible_dev_tools/resources/server/creator.py

```diff
@@ -181,3 +242,72 @@ def playbook(
         with tarfile.open(tar_file, "w:gz") as tar:
             tar.add(str(init_path), arcname=".")
         return tar_file
+
+
+class CreatorBackendV2:
```

Comment:
Why have you created another class for backend? I may be wrong, but I have the impression that you could keep a single backend class and just modify (translate) the request for it. That is because we do support only one version of the creator, the new one.